deploy/package.py

```diff
@@ -5,10 +5,14 @@
 import re
 import argparse
 import shutil
-
+import json
+from pathlib import Path
 
 # Collect args
 parser = argparse.ArgumentParser(description='Arguments required to run packaging function')
+parser.add_argument('--modes',
+    type=str, required=False, default='batch-account',
```

Comment:
how about we use nargs='+'

# This is the correct way to handle accepting multiple arguments.
# '+' == 1 or more.
# '*' == 0 or more.
# '?' == 0 or 1.
# An int is an explicit number of arguments to accept.
parser.add_argument('--nargs', nargs='+')